javaee/jaxrs-jsonb/pom.xml

```diff
@@ -47,6 +47,10 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.json</groupId>
+            <artifactId>jboss-json-api_1.1_spec</artifactId>
+        </dependency>
```

Comment:
If this is necessary, we have a bug somewhere.

Reply:
Actually this is JSON-P API? This definitely shouldn't be here.

Reply:
It is strange that the same adapter HelloWorldAdapter.java requires additional dependency in jaxrs-jsonb (org.wildfly.swarm.ts.javaee.jaxrs.jsonb )but doesn't have this dependency in javaee (org.wildfly.swarm.ts.javaee.jsonb).